astropy/units/tests/test_quantity_annotations.py

```diff
@@ -2,7 +2,6 @@
 # ruff: noqa: FA100, FA102
 
 # STDLIB
-import typing as T
 
 # THIRD PARTY
```

Comment:
I'd get rid of the `# STDLIB` and `# THIRD PARTY` comments.

Reply:
Done.